pkg/controller/stacks/job.go

```diff
@@ -254,6 +257,28 @@ func ensureDefaultVolume(volumes []corev1.Volume) []corev1.Volume {
 	return volumes
 }
 
+func ensureDefaultPodSecurityContext(psc *corev1.PodSecurityContext) *corev1.PodSecurityContext {
+	if psc != nil {
+		return psc
+	}
+
+	return &corev1.PodSecurityContext{
+		RunAsNonRoot: lo.ToPtr(true),
+	}
+}
+
+func ensureDefaultContainerSecurityContext(sc *corev1.SecurityContext) *corev1.SecurityContext {
+	if sc != nil {
+		return sc
+	}
+
+	return &corev1.SecurityContext{
+		AllowPrivilegeEscalation: lo.ToPtr(false),
+		ReadOnlyRootFilesystem:   lo.ToPtr(true),
```

Comment:
forgot to mention this in the ticket, but this is going to require configuring an emptyDir volume (which we should do of course)

Reply:
We've already had one for main `plural` working dir, but I had to add another one for `tmp` dir too.

Reply:
nice